picasso/src/test/java/com/squareup/picasso3/DispatcherTest.kt

```diff
@@ -71,13 +75,15 @@ class DispatcherTest {
   private lateinit var picasso: Picasso
   private lateinit var dispatcher: Dispatcher
 
+  private val executorService = spy(PicassoExecutorService())
   private val cache = PlatformLruCache(2048)
-  private val service = TestDelegatingService(PicassoExecutorService())
+  private val service = TestDelegatingService(executorService)
   private val bitmap1 = makeBitmap()
 
   @Before fun setUp() {
     initMocks(this)
     `when`(context.applicationContext).thenReturn(context)
+    doReturn(mock(Future::class.java)).`when`(executorService).submit(any(Runnable::class.java))
```

Comment:
Do you guys not bring in Mockito Kotlin?

Reply:
not at the moment, cuz we'd like to remove all mocks, but that will take a bit of refactoring to achieve.